pydis_site/apps/api/viewsets/bot/infraction.py

```diff
@@ -46,6 +49,7 @@ class InfractionViewSet(
     Invalid query parameters are ignored.
 
     #### Response format
+    - Response are paginated but only the actual data is returned
```

Comment:
Let's clarify what "actual data" means.
```suggestion
    The response is paginated but only the actual data is returned.
    No pagination-specific metadata is included.

```